dmapiclient/data.py

```diff
@@ -373,6 +373,7 @@ def update_user(self,
                     active=None,
                     role=None,
                     supplier_id=None,
+                    name=None,
```

Comment:
If we add the parameter here it will count as a breaking change to version 13.0.0 and need an entry in the Changelog, as any code mad enough to call this with positional parameters will need to be updated.  It is definitely neater this way, and probably existing code would be fine as it should be using named params. 

If we don't want to break things though then `name` will have to come after `updater`, right at the end of the list. That way all existing code is guaranteed to continue to work.

Either way is OK with me, up to your which you prefer.

Reply:
Ok, I changed the version to 13.0.0, interesting how some small thing can be a breaking change sometimes 😮 . But makes total sense :D.